docs/digital/cirq_interop/index.md

```diff
@@ -0,0 +1,35 @@
+# Interoperability with cirq
+
+The [cirq](https://quantumai.google/cirq) framework is a powerful tool for writing quantum circuits targeting near-term devices.
+Instead of reinventing the wheel, bloqade offers convenient interoperability with cirq that allows you to jointly use both libraries in order to develop your quantum program.
```

Comment:
```suggestion
Instead of reinventing the wheel, Bloqade offers convenient interoperability with Cirq that allows you to jointly use both libraries in order to develop your quantum program.
```